test/e2e/apps/deployment.go

```diff
@@ -130,8 +130,10 @@ var _ = SIGDescribe("Deployment", func() {
 		testProportionalScalingDeployment(f)
 	})
 	ginkgo.It("should not disrupt a cloud load-balancer's connectivity during rollout", func() {
-		e2eskipper.SkipUnlessNodeCountIsAtLeast(2)
 		e2eskipper.SkipUnlessProviderIs("aws", "azure", "gce", "gke")
+		nodes, err := e2enode.GetReadySchedulableNodes(c)
+		framework.ExpectNoError(err)
+		e2eskipper.SkipUnlessAtLeast(len(nodes.Items), 3, "load-balancer test requires at least 3 schedulable nodes")
```

Comment:
we don't expect this to happen in fips tests.  I was thinking more of a straight fail

Reply:
the fact that somehow fips test ends with 5 nodes and not 6 is not the business of this test, if the required minimum is not here we skip, since that's the only way we can get it upstream. i don't want openshift specific carry

Reply:
> the fact that somehow fips test ends with 5 nodes and not 6 is not the business of this test, if the required minimum is not here we skip, since that's the only way we can get it upstream. i don't want openshift specific carry

while that may be true, the immediate problem is going to boomerang straight back to you over and over and over again unless you can provide a clear signal.  skipping this test isn't a win, it's a loss.